site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -98,3 +98,64 @@ Use:
 
 You can also change the container image with `--image` and args with `--args`.
 For more customization, you can edit the example script.
+
+### Interact with Queues and Jobs
+
+If you are developing an application that submits jobs and needs to interact
+with and check on them, you likely want to interact with queues or jobs directly.
+After running the example above, you can test the following example to interact
+with the results. Write the following to a script called `sample-queue-control.py`.
+
+{{% include "python/install-kueue-queues.py" "python" %}}
```

Comment:
Should this be `sample-queue-control.py`?

Is there a file called `sample-queue-control.py`?